tests/unit-tests/data_utils.py

```diff
@@ -156,6 +162,66 @@ def generate_dummy_dist_graph(dirname, size='tiny', graph_name='dummy'):
     return partion_and_load_distributed_graph(hetero_graph=hetero_graph, dirname=dirname,
                                               graph_name=graph_name)
 
+def create_lm_graph(tmpdirname):
```

Comment:
can you add docstring to explain the difference between the two LM graphs?

Reply:
Updated.